a4kSubtitles/download.py

```diff
@@ -7,6 +9,10 @@ def __download(core, filepath, request):
             core.shutil.copyfileobj(r.raw, f)
 
 def __extract_gzip(core, archivepath, filename):
+    exts = subtitles_exts + subtitles_exts_secondary
+    if not any(filename.lower().endswith(ext) for ext in exts):
+        filename = filename + ".srt"
```

Comment:
Adding `.srt` on all files which do not have one, may lead to issues. If the extension addition is to be implemented we need to have logic to detect the different subtitle formats: https://matesub.com/resources/subtitle-file-formats

Reply:
I've mostly seen `.srt`, `.ass` and `.sub`, but don't think there are limitations in the subtitle providers for formats. KODI does detect it from content and ext is not required. If we add it here just for KODI's auto loading in some cases, I'm fine with it. Just add a comment with the intentions to point out that this is just for marking it as sub file and not really the known format.